src/core_plugins/elasticsearch/lib/__tests__/create_proxy.js

```diff
@@ -1,5 +1,5 @@
 import expect from 'expect.js';
-import createProxy from '../create_proxy';
+import { createProxy } from '../create_proxy';
```

Comment:
Should be importing `createPath` here and update the references for the tests to pass.